src/plugins/pluginManager/plugin.ts

```diff
@@ -134,6 +135,24 @@ export class PluginManagerPlugin extends ProposalPlugin<
     }
   }
 
+  public decodeDataByPluginName = (pluginName: string, encodedData: string) => {
+    let abi
```

Comment:
hope about  `if pluginName ===NULL return empty object `?

Reply:
Since plugin name is a string I would say: 
`if pluginName === '' return {}`